packages/vite/src/node/utils.ts

```diff
@@ -452,3 +455,7 @@ export function combineSourcemaps(
 
   return map as RawSourceMap
 }
+
+export function unique<T>(arr: T[]): T[] {
+  return Array.from(new Set(arr))
```

Comment:
**question:** is `Array.from(new Set(arr))` faster then `[...new Set(arr)]`?

Reply:
I don't know, but so far we don't need `unique` to be performant; the dependency list wouldn't get that big that it would matter.